### ISSUE TITLE
Ability to set the default domain in the dashboard

### ISSUE THREAD

**Issue by shoulders** [virtualmin/virtualmin-gpl#802]
|  SYSTEM INFORMATION||
|----------------------|---------------------------|
|  OS type and version |     Ubuntu Linux 22.04.4     |
|  Webmin version      |     2.111      |
|  Usermin version     |     2.010     |
|  Virtualmin version  |     7.10.0       |
|  Theme version       |     21.10      |
|  Package updates     |      3  package updates are available   |

## The background

When ever I login there is the domain that is pre-selected is always the same because it is first in the alphabet and this is annoying, but not a show stopper.

![image](https://github.com/virtualmin/virtualmin-gpl/assets/319997/6b30fc46-d646-4f1b-8022-a5ab2cd795ba)

This is not the same as `Virtualmin --> Web Configuration --> Website Options --> Default website for IP address`

## Proposed solution

Have the option to select which is the default domain for Virtulamin

 `Virtualmin --> Web Configuration --> Website Options --> Default website for Virtualmin dashboard`

![image](https://github.com/virtualmin/virtualmin-gpl/assets/319997/8f3f3653-955e-4155-bf18-1678cabdd77f)

**Comment by jcameron** [comment]
There should be an icon in the top-right corner of the dashboard that opens a preferences page where you can set the default domain to be selected when you login.

**Comment by shoulders** [comment]
This is all that I can see and I have looked through and I cant find any option. There is no config icon.

![image](https://github.com/virtualmin/virtualmin-gpl/assets/319997/effe96f1-c22c-4ee5-81de-a22e275354ba)

**Comment by iliajie** [comment]
That's in the framed theme, whereas in Authentic theme you should click a palette button in the bottom of navigation menu, to get to theme configuration page.

**Comment by shoulders** [comment]
I found the option **Default page for Virtualmin** and this gives 3 options:

- Dashboard
- Virtualmin Virtual Servers
- Virtual Server Summary + your selected domain

What my thoughts were to select the page you want with the `Default page for Virtualmin`, and then select your preferred domain. i.e. example.com with another option.

For me this is useful on my dev server because I am always messing and it is good for screenshots etc..

**Comment by iliajie** [comment]
Ah, I don't know. It sounds like a lot of work; I'm not really sure we want to implement this.

If I understood correctly, you'd want to have a domain menu loaded for the domain `example.com` while the right page is opened to, let's say, **PHP Options**?

**Comment by shoulders** [comment]
yes that is correct, but I only would want the dashboard, and Virtual Servers Summary, and single virtual server summary

The code that currently handles `Default page for Virtualmin` could be split without massive reworking (this a guess)

![image](https://github.com/virtualmin/virtualmin-gpl/assets/319997/da7ff2df-2f0b-4526-9104-1347811530f7)

**Comment by iliajie** [comment]
Well, we can keep this ticket open, and I may implement this sometime later, but it is not a priority for now, and pretty complicated to implement to be honest.

**Comment by iliaross** [comment]
Reopened, as this was closed by an automated cleanup run. @shoulders, sorry for the noise, and thanks for the contribution. We definitely didn't mean any disrespect.

If it turns out the issue is already resolved for you, you're welcome to close it.
